AlamofireObjectMapper.podspec

```diff
@@ -1,22 +1,22 @@
 Pod::Spec.new do |s|
 
   s.name = "AlamofireObjectMapper"
-  s.version = "6.0.0"
+  s.version = "6.0.1"
   s.license = { :type => "MIT", :file => "LICENSE" }
   s.summary = "An extension to Alamofire which automatically converts JSON response data into swift objects using ObjectMapper"
-  s.homepage = "https://github.com/tristanhimmelman/AlamofireObjectMapper"
+  s.homepage = "https://github.com/tirrorex/AlamofireObjectMapper"
   s.author = { "Tristan Himmelman" => "tristanhimmelman@gmail.com" }
-  s.source = { :git => 'https://github.com/tristanhimmelman/AlamofireObjectMapper.git', :tag => s.version.to_s }
+  s.source = { :git => 'https://github.com/tirrorex/AlamofireObjectMapper.git', :tag => s.version.to_s }
```

Comment:
Why did you change this URL?

Reply:
Forgot about that before doing the pr.
I obviously changed to url to be able to use my code as the source in my project.